### PR TITLE
Separate pipeline summary from projection and reconstruction

### PR DESCRIPTION
**Pull request by dreycenfoiles**
Currently, the Azure pipeline will compute projections, reconstruction, and summarize benchmarking results all within the same job. This means that all the algorithms aren't being compared against each other. This pull request changes the pipeline so that all reconstruction results will be aggregated together before summarization.

### CODE REVIEW

**Review by carterbox — COMMENTED** [review]
Why are you using pip to install pytest instead of conda? You didn't explain this change anywhere.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -16,220 +16,313 @@ trigger:
...
+            --trials 4 \
+            --width 400 \
+            --num-angles 50 \
+            --output-dir $outputDir
+
+          cd tomopy.github.io
+          git config --global user.name "Dreycen Foiles"
+          git config --global user.email "foilesdreycen@gmail.com"
+          git add *
+          git commit -m "Bi-monthly benchmark"
```
Comment (carterbox) [inline azure-pipelines.yml]:
```suggestion
          git commit -m "Simulate data for benchmark"
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -16,220 +16,313 @@ trigger:
...
+          --ncore 4 \
+          --max-iter 50 \
+          --output-dir $outputDir \
+          --algorithm $(algorithmName)
+
+        cd tomopy.github.io
+        git config --global user.name "Dreycen Foiles"
+        git config --global user.email "foilesdreycen@gmail.com"
+        git add *
+        git commit -m "Bi-monthly benchmark"
```
Comment (carterbox) [inline azure-pipelines.yml]:
```suggestion
        git commit -m "Bi-monthly benchmark $(algorithmName)"
```

File: azure-pipelines.yml [hunk 3]
```diff
@@ -16,220 +16,313 @@ trigger:
...
+        python -Om benchmarking.summarize \
+              --phantom peppers \
+              --trials 4 \
+              --output-dir $outputDir
+
+        cd tomopy.github.io
+        git config --global user.name "Dreycen Foiles"
+        git config --global user.email "foilesdreycen@gmail.com"
+        git add *
+        git commit -m "Bi-monthly benchmark"
```
Comment (carterbox) [inline azure-pipelines.yml]:
```suggestion
        git commit -m "Benchmark data summary"
```

File: azure-pipelines.yml [hunk 4]
```diff
@@ -16,220 +16,313 @@ trigger:
...
+          --ncore 4 \
+          --max-iter 50 \
+          --output-dir $outputDir \
+          --algorithm $(algorithmName)
+
+        cd tomopy.github.io
+        git config --global user.name "Dreycen Foiles"
+        git config --global user.email "foilesdreycen@gmail.com"
+        git add *
+        git commit -m "Bi-monthly benchmark"
```
Comment (carterbox) [inline azure-pipelines.yml]:
```suggestion
        git commit -m "Bi-monthly benchmark $(algorithmName)"
```

**Comment by dreycenfoiles** [comment]
> Why are you using pip to install pytest instead of conda? You didn't explain this change anywhere.

I left it out of the environment yaml file because the other steps don't need it, so it felt inefficient to always install it. I didn't have any reason for using pip over conda when installing, though. Is conda preferred?

**Comment by dreycenfoiles** [comment]
It appears that when installing pytest with conda, the installation process hangs and pytest won't install.

**Review by carterbox — APPROVED** [review]

**Pull request merged**